### PR TITLE
Ask Survey embed via model attribute

### PR DESCRIPTION
**Pull request by cadecairos**
This is just a proof of concept. needs more thought+development

sample survey id: `5a5fb1f70719a40001f03e65`

### CODE REVIEW

File: source/pug/views/opportunity.pug [hunk 1]
```diff
@@ -35,6 +35,9 @@ block content
       div(class!=`{% if page.parent.title != "Opportunity" or page.children.all.count > 0 %}col-12 col-md-9{% else %}col-12 col-lg-8 offset-lg-2{% endif %}`)
         .row
           .cms.col-12 {{ page.landingpage.content | safe }}
+        .row
+          .cms.col-12
+            iframe(width=`100%`, height=580 src=`https://survey.mofoprod.net/{{ page.landingpage.survey_id }}.html`)
```
Comment (alanmoo) [inline source/pug/views/opportunity.pug]: This is always going to render even if there's no survey, isn't it?

Comment (cadecairos) [reply source/pug/views/opportunity.pug]: oh yeah. I had a conditional when I didn't realize this was a pug template. forgot to bring it over

**Comment by mmmavis** [comment]
I don't think I know enough of this codebase as well as Django to review this PR, but I replaced `{{ page.landingpage.survey_id }}` with the sample survey id and the embed wasn't showing.

screen shot 2018-01-19 at 2 39 50 pm

**Comment by cadecairos** [comment]
You're going to have to update the CSP_CHILD_SRC environment variable of your dev environment so that the blocked domain is allowed by the Content Security Policy (`https://survey.mofoprod.net`)

**Pull request closed without merging**